src/iio_wrapper.cpp

```diff
@@ -693,3 +693,101 @@ int32_t IIOWrapper::decimation_filter()
 
     return valuel;
 }
+
+// --------------------------------------------------------------------------
+
+void IIOWrapper::set_burst_size_selection(int32_t val)
+{
+    long long valuel = val;
+    iio_device_debug_attr_write_longlong(m_dev,"burst_size_selection", valuel);
+}
+
+void IIOWrapper::set_burst_data_selection(int32_t val)
+{
+    long long valuel = val;
+    iio_device_debug_attr_write_longlong(m_dev,"burst_data_selection", valuel);
+}
+
+void IIOWrapper::set_linear_acceleration_compensation(int32_t val)
+{
+    long long valuel = val;
+    iio_device_debug_attr_write_longlong(m_dev,"linear_acceleration_compensation", valuel);
+}
+
+void IIOWrapper::set_point_of_percussion_alignment(int32_t val)
+{
+    long long valuel = val;
+    iio_device_debug_attr_write_longlong(m_dev,"point_of_percussion_alignment", valuel);
+}
+
+void IIOWrapper::set_internal_sensor_bandwidth(int32_t val)
+{
+    long long valuel = val;
+    iio_device_debug_attr_write_longlong(m_dev,"internal_sensor_bandwidth", valuel);
+}
+
+void IIOWrapper::set_sync_mode_select(int32_t val)
+{
+    long long valuel = val;
+    iio_device_debug_attr_write_longlong(m_dev,"sync_mode_select", valuel);
+}
+
+void IIOWrapper::set_sync_polarity(int32_t val)
+{
+    long long valuel = val;
+    iio_device_debug_attr_write_longlong(m_dev,"sync_polarity", valuel);
+}
+
+void IIOWrapper::set_data_ready_polarity(int32_t val)
+{
+    long long valuel = val;
+    iio_device_debug_attr_write_longlong(m_dev,"data_ready_polarity", valuel);
+}
+
+void IIOWrapper::set_sync_signal_scale(int32_t val)
+{
+    long long valuel = val;
+    iio_device_debug_attr_write_longlong(m_dev,"sync_signal_scale", valuel);
+}
+
+void IIOWrapper::set_decimation_filter(int32_t val)
+{
+    long long valuel = val;
+    iio_device_debug_attr_write_longlong(m_dev,"decimation_filter", valuel);
+}
+
+void IIOWrapper::set_accel_calibbias_x(int32_t val)
+{
+    long long valuel = val;
+    iio_channel_attr_write_longlong(m_channel_accel_x, "calibbias", valuel);
+}
+
+void IIOWrapper::set_accel_calibbias_y(int32_t val)
+{
+    long long valuel = val;
+    iio_channel_attr_write_longlong(m_channel_accel_y, "calibbias", valuel);
+}
+
+void IIOWrapper::set_accel_calibbias_z(int32_t val)
+{
+    long long valuel = val;
+    iio_channel_attr_write_longlong(m_channel_accel_z, "calibbias", valuel);
+}
+
+void IIOWrapper::set_anglvel_calibbias_x(int32_t val)
+{
+    long long valuel = val;
+    iio_channel_attr_write_longlong(m_channel_anglvel_x, "calibbias", valuel);
+}
+
+void IIOWrapper::set_anglvel_calibbias_y(int32_t val)
+{
+    long long valuel = val;
+    iio_channel_attr_write_longlong(m_channel_anglvel_y, "calibbias", valuel);
+}
+
+void IIOWrapper::set_anglvel_calibbias_z(int32_t val)
+{
+    long long valuel = val;
+    iio_channel_attr_write_longlong(m_channel_anglvel_z, "calibbias", valuel);
+}
```

Comment:
all of these should return what iio_channel_attr_write_longlong, which is 0 in case of success and error code otherwise, if an error occurs the user should know about it

Reply:
done